addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -66,20 +69,15 @@ $link-hover-color: auto-contrast(darken($link-color, 15%), $body-bg, 'o-cc1-link
 $link-decoration: if(o-website-value('link-underline') == 'always', underline, none) !default;
 $link-hover-decoration: if(o-website-value('link-underline') != 'never', underline, none) !default;
 
-// Table
-$table-striped-bg-factor: .01 !default;
-$table-cell-padding-x: .75rem !default;
-$table-cell-padding-y: .75rem !default;
-$table-cell-padding-x-sm: .3rem !default;
-$table-cell-padding-y-sm: .3rem !default;
-
 // Grid
 $grid-gutter-width: 30px !default;
 
 // Components
 //
 // Define common padding and border radius sizes and more.
 
+$border-color: var(--o-border-color, #{o-color('300')}) !default;
```

Comment:
Add a note saying that the fallback value should always match the  '--o-border-color' default value in `addons/website/static/src/scss/website.scss`